server/service/src/requisition/request_requisition/update/mod.rs

```diff
@@ -88,6 +92,20 @@ pub fn update_request_requisition(
         )
     );
 
+    if input.status == Some(UpdateRequestRequistionStatus::Sent) {
+        log_entry(
+            &ctx.connection,
+            &LogRow {
+                id: uuid(),
+                log_type: LogType::RequisitionStatusSent,
+                user_id: None,
```

Comment:
is it possible to add the user?

Reply:
Will the user who sends the requisition be different from the one who makes it? 🤔

Reply:
potentially, yup. the status change can be made by anyone with permission

Reply:
I can add the user once they're added to ServiceContext... need it for the deletions as well... See Clemen's comment below 😄